cli/flox-rust-sdk/src/providers/buildenv.rs

```diff
@@ -284,6 +285,12 @@ impl BuildEnvNix {
             return Ok(());
         }
 
+        let _span = info_span!(
+            "build from catalog",
+            progress = format!("Building '{}' from source", locked.install_id)
```

Comment:
Should this be the pkg-path instead, as that's what is actually being built to realise the install-id?

Reply:
yes, likely
we print the flake installable for flakes as well.